docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,23 +1,25 @@
-[[cxf-extension-for-quarkus]]
+[[quarkus-cxf-index]]
 = {quarkus-cxf-project-name}
 
 This project aims at porting http://cxf.apache.org/[Apache CXF] to https://quarkus.io/[Quarkus] runtime
 including full support for producing https://quarkus.io/container-first/[GraalVM native images].
 
 [IMPORTANT]
+====
 While http://cxf.apache.org/[Apache CXF] implements wide range of standards and specifications,
 {quarkus-cxf-project-name} is currently focusing *only* on supporting the ones related to
 *Simple Object Access Protocol* (https://en.wikipedia.org/wiki/SOAP[SOAP])
 and *Java API for XML Web Services* (https://en.wikipedia.org/wiki/JAX-WS[JAX-WS]).
-Refer to xref:reference/index.adoc[Reference] to learn which parts of CXF are supported.
+Refer to xref:reference/index.adoc#reference-index[Reference] to learn which parts of CXF are supported.
+====
 
 The documentation is divided into the following main parts:
 
-* xref:user-guide/index.adoc[User guide] - explains the typical usage of {quarkus-cxf-project-name}
-* xref:security-guide/index.adoc[Security guide] - documents various security related aspects of {quarkus-cxf-project-name}, such as SSL/TLS, Authentication and Authorization
-* xref:release-notes/index.adoc[Release notes] - Release notes documents for the past releases
-* xref:release-planning.adoc[Release planning] - when new releases of {quarkus-cxf-project-name} are to be expected
-* xref:reference/index.adoc[Reference] - the full list of supported specifications, use cases and configuration options
+* xref:user-guide/index.adoc#user-guide-index[User guide] - explains the typical usage of {quarkus-cxf-project-name}
+* xref:security-guide/index.adoc#security-guide-index[Security guide] - documents various security related aspects of {quarkus-cxf-project-name}, such as SSL/TLS, Authentication and Authorization
+* xref:release-notes/index.adoc#release-notes-index[Release notes] - Release notes documents for the past releases
+* xref:release-planning.adoc#release-planning[Release planning] - when new releases of {quarkus-cxf-project-name} are to be expected
+* xref:reference/index.adoc#reference-index[Reference] - the full list of supported specifications, use cases and configuration options
```

Comment:
I see why these `*-index` anchors are needed. We break the external links, but it is perhaps not that important in this specific case. Hence I am fine with the change.